ts/src/bitget.ts

```diff
@@ -2549,13 +2549,23 @@ export default class bitget extends Exchange {
         /**
          * @method
          * @name bitget#createOrder
+         * @see https://bitgetlimited.github.io/apidoc/en/spot/#place-order
```

Comment:
@Dan-krm also upon stumble on a method without proper documentation, please update it